### ISSUE TITLE
Accordions multi-select ARIA change

### ISSUE THREAD

**Issue by Mattcessability** [ArtisticAbode/Accessible-Tabs-and-Accordions#1]
Hi ArtisticAbode,

I've noticed that the multi-select accordions are using aria-selected rather than aria-expanded on the accordion button, and aria-expanded has instead been placed on the expanded content. See your code below:

`<button class="toggle" aria-selected="true" aria-controls="accordion-tab-13" id="accordion-13" type="button">First Heading</button>`
`<div class="content" aria-expanded="true" aria-hidden="false" aria-labelledby="accordion-13" id="accordion-tab-13" tabindex="-1">`
`<p>Content for the first accordion. This can include multiple paragraphs, nested headings, lists, images and even additional tabs and accordions.</p>`
`</div>`

I've only tested this with one screen reader so far (NVDA), but my screen reader isn't announcing that the section is expanded OR selected at all. All it states is "First accordion, button" and doesn't announce any extra information. I find things work much better for these type of accordions if aria-selected is not used at all, and that aria-expanded is moved to the `<button>`, like so:

`<button class="toggle" aria-expanded="true" aria-controls="accordion-tab-13" id="accordion-13" type="button">First Heading</button>`
`<div class="content" aria-hidden="false" aria-labelledby="accordion-13" id="accordion-tab-13" tabindex="-1">`
`<p>Content for the first accordion. This can include multiple paragraphs, nested headings, lists, images and even additional tabs and accordions.</p>`
`</div>`

Using this method, NVDA now states "First accordion button expanded", which makes much more sense for an accordion.

Examples from the W3C that place aria-expanded on the `<button>`:

- https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html
- https://www.w3.org/WAI/GL/wiki/Using_the_WAI-ARIA_aria-expanded_state_to_mark_expandable_and_collapsible_regions

Can your team please look into this and see if it should be changed? All of the other examples on the page seem perfect, but this one has been bugging me.

Cheers,

Matthew Putland

**Comment by ArtisticAbode** [comment]
Hi Matthew, 

Thanks for the feedback. Someone else also mentioned this and a couple other things that I have updated now in the code. 

Cheers,
Andrea
